consensus/hotstuff/validator/validator_test.go

```diff
@@ -303,9 +303,9 @@ func (vs *VoteSuite) TestVoteMismatchingView() {
 
 func (vs *VoteSuite) TestVoteSignatureError() {
 
-	// make the verification fail on signature
+	// make the verification fail on exception
 	*vs.verifier = mocks.Verifier{}
-	vs.verifier.On("VerifyVote", vs.signer, vs.vote.SigData, vs.block).Return(true, errors.New("dummy error"))
+	vs.verifier.On("VerifyVote", vs.signer, vs.vote.SigData, vs.block).Return(fmt.Errorf("some exception"))
 
 	// check that the vote is no longer validated
 	_, err := vs.validator.ValidateVote(vs.vote, vs.block)
```

Comment:
Can we add a check that we really get an exception here? I mean validator possibly can wrap it into sentinel error, need to check this.

Reply:
I think [Yurii's suggestion](https://github.com/onflow/flow-go/pull/1741/files#r766680994) is to add the following assertion at the end of this test: 
```golang
assert.False(vs.T(), model.IsInvalidVoteError(err), "internal exception should not be interpreted as invalid vote")
```